### PR TITLE
renamed brand

### PR DESCRIPTION
**Pull request by awemulya**
## Checklist

1. [ ] If you've added code that should be tested, add tests
2. [ ] If you've changed APIs, update (or create!) the documentation
3. [ ] Ensure the tests pass
4. [ ] Make sure your code lints and you followed [our coding style](https://github.com/kobotoolbox/kpi/blob/master/CONTRIBUTING.md)
5. [ ] If this is a big feature, make sure to prefix the title with `Feature:` and add a thorough description for non-dev folk

### CODE REVIEW

File: kpi/templates/base.html [hunk 1]
```diff
@@ -13,7 +13,7 @@
 <link href="//netdna.bootstrapcdn.com/font-awesome/4.0.3/css/font-awesome.css" rel="stylesheet">
 <meta charset="utf-8">
 <meta http-equiv="X-UA-Compatible" content="IE=edge">
-<title>KoBoToolbox {% block title %}{% endblock %}</title>
+<title>NobiForms {% block title %}{% endblock %}</title>
```
Comment (jd-alexander) [inline kpi/templates/base.html]: Hey @awemulya there should be a space. It's called **Nobi Forms**.

File: kpi/templates/registration/registration_form.html [hunk 2]
```diff
@@ -53,9 +53,11 @@ <h1>{% trans "Create an account" %}</h1>
 
   <div class="registration__second-half">
       {% if welcome_message %} {{ welcome_message|safe }} {% else %}
-      <p>KoBoToolbox is an integrated set of tools for building forms and collecting interview responses. It is built by the Harvard Humanitarian Initiative for easy and reliable use in difficult field settings, such as humanitarian emergencies or
-          post-conflict environments. <a href="http://www.kobotoolbox.org">Read more</a></p>
-      <p>You can download and install KoBoToolbox on <a href="http://support.kobotoolbox.org/customer/portal/articles/1691108-install-kobotoolbox-on-your-own-server">your own server</a> or <a href="http://support.kobotoolbox.org/customer/portal/articles/1691105-using-kobotoolbox-offline">on your own computer</a>.</p>
+      <p>NobiForms is an integrated set of tools for building forms and collecting interview responses. It is built by the Harvard Humanitarian Initiative for easy and reliable use in difficult field settings, such as humanitarian emergencies or
```
Comment (jd-alexander) [inline kpi/templates/registration/registration_form.html]: Oh I never remember this text was here 😄 Put this there sir!

"Nobi Forms is a dynamic mobile form solution that revolutionizes how data is collected, measured, analysed and shared. Our data collection suite of web, mobile and desktop applications allow for unprecedented decision-making capabilities and improved efficiency within an organisation."

Comment (awemulya) [reply kpi/templates/registration/registration_form.html]: sure !!

File: kpi/templates/base.html (hunk 1 above)
Comment (jd-alexander) [reply kpi/templates/base.html]: @raeesaroj remember this one! 😄

**Pull request merged**